browser/brave_profile_prefs.cc

```diff
@@ -339,6 +340,10 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(kShowWalletIconOnToolbar, true);
 #endif
 
+  // Brave Search
+  registry->RegisterListPref(brave_search::prefs::kDailyAsked);
```

Comment:
these should go in `BraveSearchHost::RegisterProfilePrefs` or similar and call that here instead

Reply:
fixed